.github/workflows/pr-conventional-commits.yaml

```diff
@@ -52,4 +52,4 @@ jobs:
       env:
         PR_BASE_REF: ${{ github.event.pull_request.base.ref }}
         PR_HEAD_REF: ${{ github.event.pull_request.head.ref }}
-        PR_HEAD_REPO_CLONE_URL: ${{ github.event.pull_request.head.repo.clone_url }}
+        PR_HEAD_REPO_CLONE_URL: ${{ github.event.pull_request.head.repo.clone_url }}
```

Comment:
Don't really know what happened here but the file was no modified.

Reply:
Does your editor have last-line linefeed settings?

Reply:
Ah: https://github.com/jenstroeger/python-package-template/actions/runs/3474640735/jobs/5808048530#step:10:71

Did the git hooks not run locally on commit and push?